projects/storefrontlib/ng-package.json

```diff
@@ -9,7 +9,8 @@
       "@ngrx/store": "store",
       "@ngrx/router-store": "fromNgrxRouter",
       "@ng-select/ng-select": "ngSelect",
-      "@spartacus/core": "core"
+      "@spartacus/core": "core",
+      "ngx-infinite-scroll": "ngxInfiniteScroll"
```

Comment:
fix for `No name was provided for external module 'ngx-infinite-scroll' in output.globals – guessing 'ngxInfiniteScroll'`